lib/routes/dedao/articles.ts

```diff
@@ -0,0 +1,128 @@
+import { Route } from '@/types';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+
+export const route: Route = {
+    path: '/articles/:id?', // 添加 id 参数，用于控制 pid
+    categories: ['new-media'],
+    example: '/articles/9', // 示例路径更新
+    parameters: { id: '文章类型 ID，8 为得到头条，9 为得到精选，默认为 8' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['igetget.com'],
+            target: '/articles/:id',
+        },
+    ],
+    name: '得到文章',
+    maintainers: ['Jacky-Chen-Pro'],
+    handler,
+    url: 'https://www.igetget.com',
+};
+
+// 提取文章内容的函数
+function extractArticleContent(data) {
+    const paragraphs = [];
+
+    for (const block of data) {
+        if (block.type === 'paragraph') {
+            let paragraph = '';
+            for (const item of block.contents) {
+                if (item.type === 'text' && item.text?.content) {
+                    let content = item.text.content;
+                    if (item.text?.highlight) {
+                        content = `<strong>${content}</strong>`;
+                    }
+                    paragraph += content;
+                }
+            }
+            if (paragraph) {
+                paragraphs.push(`<p>${paragraph}</p>`);
+            }
+        }
+    }
+    return paragraphs.join('');
+}
```

Comment:
You need a recursive function
```ts
function extractArticleContent(data) {
    if (!data || typeof data !== 'object') {
        return '';
    }

    let html = '';

    switch (data.type) {
        case 'paragraph':
            html += '<p>';
            if (data.contents && Array.isArray(data.contents)) {
                for (const item of data.contents) {
                    html += extractArticleContent(item);
                }
            }
            html += '</p>';
            break;

        case 'text':
            if (data.text) {
                let content = data.text.content || '';
                if (data.text.bold || data.text.highlight) {
                    content = `<strong>${content}</strong>`;
                }
                html += content;
            }
            break;

        default:
            break;
    }

    return html;
}
```
and call it in
```ts
JSON.parse(articleData.articleContent.content).map((data) => extractArticleContent(data)).join('')
```

Please render the `image` and `hr` as well.